SampleCode/SwiftSampleCode/DronecodeSDKSwiftDemo/Controllers/CameraViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CameraViewController.swift
+//  DronecodeSDKSwiftDemo
+//
+//
+
+import Foundation
+
+import UIKit
+import Dronecode_SDK_Swift
+
+class CameraViewController: UIViewController{
+    
+    // MARK: - Properties
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    lazy var video: RTSPPlayer = createRTSPVideo()
```

Comment:
So that's the actual player.

Reply:
I would choose VLCKit, really easier to use. 
RTSPPlayer is a light version of https://github.com/durfu/DFURTSPPlayer.
I'm waiting the test of @julianoes before merging with E90